lib/configuration.yaml

```diff
@@ -14861,20 +14861,20 @@ testnet_full: &testnet_full
 
   update: &testnet_full_update
     applicationName: cardano-sl
-    applicationVersion: 0
+    applicationVersion: 1
     lastKnownBlockVersion:
       bvMajor: 0
-      bvMinor: 0
+      bvMinor: 1
       bvAlt: 0
 
 testnet_wallet: &testnet_wallet
   <<: *testnet_full
   update: &testnet_wallet_update
     applicationName: csl-daedalus
-    applicationVersion: 0
```

Comment:
All good then, thanks!

Reply:
So we have no option but to downgrade 2.0.0RC2 to 1.3.2 on the testnet, indeed.

Reply:
I think this kind of a downgrade is a first of its kind..

Reply:
Yes, a downgrade would not normally work because the DB is not backwards-compatible.
Since we have not yet announced installer links, we only need to advise our small gang of testnet guinea pigs to clear out their Daedalus state and install this version.
All that matters for the update system is that the applicationVersion increases.